photon-colors/photon-colors.gpl

```diff
@@ -0,0 +1,67 @@
+GIMP Palette
+Name: Photon Colors
+# Photon Colors GPL Color Palette v1.4.0
+
+255 26 217 Magenta 50
```

Comment:
We're picking up the GIMP colour renaming here, too? (I think that's okay, but wanted to make sure it was on purpose.)

Reply:
Yes, this is on purpose ☺️